Mathlib/RingTheory/Nakayama.lean

```diff
@@ -141,8 +144,37 @@ theorem le_of_le_smul_of_le_jacobson_bot {R M} [CommRing R] [AddCommGroup M] [Mo
 [Stacks 00DV](https://stacks.math.columbia.edu/tag/00DV).
 See also `sup_smul_eq_sup_smul_of_le_smul_of_le_jacobson` for a generalisation
 to the `jacobson` of any ideal -/
+@[stacks 00DV "(4)"]
 theorem smul_le_of_le_smul_of_le_jacobson_bot {I : Ideal R} {N N' : Submodule R M} (hN' : N'.FG)
     (hIJ : I ≤ jacobson ⊥) (hNN : N' ≤ N ⊔ I • N') : I • N' ≤ N :=
   smul_le_right.trans (le_of_le_smul_of_le_jacobson_bot hN' hIJ hNN)
 
+open Pointwise
+
+@[stacks 00DV "(3)"]
```

Comment:
This is only the first part of 00DV(3), right?
Because there is also the claim about localizations in https://stacks.math.columbia.edu/tag/00DV

Reply:
Strictly speaking yes, although the second claim does not really have anything to do with Nakayama. I added a (slightly generalized) version of it in a different file anyway.